frontend/app/src/store/session/visibility.ts

```diff
@@ -4,7 +4,7 @@ import { type Pinned } from '@/store/session/types';
 export const useAreaVisibilityStore = defineStore('session/visibility', () => {
   const showAbout = ref(false);
   const pinned = ref<Nullable<Pinned>>(null);
-  const showDrawer = ref(false);
+  const showDrawer = ref(true);
```

Comment:
This change is why the e2e tests fail. 

The default should still be false since we should not show the drawer by default in smaller windows.

What we should do is automatically show it in bigger layouts, so I would put this change in the default layout.

Reply:
should be updated now